keadm/README.md

```diff
@@ -1,17 +1,263 @@
-# KubeEdge Installer
+# Getting Started with KubeEdge Installer
 
-[Kubeedge Installer Doc](<https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/kubeedgeadm-scope.md/>)
+Please refer to KubeEdge Installer proposal document for details on the motivation of having KubeEdge Installer.
+It also explains the proposed functionality of the proposed commands.
+[KubeEdge Installer Doc](<https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/kubeedgeadm-scope.md/>)
 
-#### Build KubeEdgeinstaller
+## Limitation
 
- ```shell
-    cd $GOPATH/src/github.com/kubeedge/kubeedge/keadm
-    
-    make
-     
-    sudo mv keadm /usr/local/bin
- ```
-  
-#### Execute command to get started
+- Currently support of `KubeEdge installer` is available only for Ubuntu OS. CentOS support is in-progress.
 
-sudo keadm --help
\ No newline at end of file
+## Downloading KubeEdge Installer
+
+1. Goto [KubeEdge Release](<https://github.com/kubeedge/kubeedge/releases>) page and download `keadm-v0.3.0-beta.0-linux-amd64.tar.gz`.
```

Comment:
For now , its v0.3.0-beta0, but in future it would be 1.0, so better rename to keadm-<release_version>-linux-<arch>.tar.gz and mention that currently only amd64 is supported